src/components/card/expanded/EmendasInfo.vue

```diff
@@ -71,31 +95,64 @@ export default {
       }
     },
     orderedEmendas () {
-      const result = this.emendas[this.id].filter(function (emenda) { return emenda.distancia !== -1 })
+      const result = this.emendas[this.id].filter(function (emenda) {
+        return emenda.distancia !== -1
+      })
       return result.sort((a, b) => b.distancia - a.distancia)
     },
     getDiscrepantes () {
-      return _.take(this.orderedEmendas, Math.min(5, _.ceil(this.orderedEmendas.length / 2)))
+      return _.take(
+        this.orderedEmendas,
+        Math.min(5, _.ceil(this.orderedEmendas.length / 2))
+      )
     },
     getSemelhantes () {
-      return _.reverse(_.takeRight(this.orderedEmendas, Math.min(5, _.floor(this.orderedEmendas.length / 2))))
+      if (!this.verificaSeMostraEmendasAparentes) {
+        return _.reverse(
+          _.takeRight(
+            this.orderedEmendas,
+            Math.min(5, this.orderedEmendas.length)
+          )
+        )
+      } else {
```

Comment:
Poderia inicializar uma variavel com this.orderedEmendas e realizar um if para this.verificaSeMostraEmendasAparentes atualizando a variavel para _.floor(this.orderedEmendas.length / 2) e só chamar uma vez o reverse no código